### PR TITLE
Script for generation of new Hamilton file (Buffer exchange TWIST)

### PR DESCRIPTION
**Pull request by Karl-Svard**
### Added
New EPP script for generating Hamilton csv-file in step Buffer exchange TWIST. 

### Fixed

**Steps to consider while deploying**
- Configuration changes:
- Documentation updates:
- Inform users by email:

### Review:
- [ ] Code approved by
- [ ] Tests executed on stage by:  (Document the test done with screen shots and description.)
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: tests/EPPs/files/test_buffer_exchange_twist_file.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                 Artifact(lims, id="2-2336705"),
+                 Artifact(lims, id="2-2336706"),
+                 Artifact(lims, id="2-2336707")]
+
+    # WHEN running get_file_data_and_write
+    get_file_data_and_write(artifacts, file)
+
+    # THEN the file has been created and the file content is as expected.
+    assert file.read_text() == hamilton_buffer_exchange
+    # remove file afterwards?
```
Comment (Karl-Svard) [inline tests/EPPs/files/test_buffer_exchange_twist_file.py:29]: @mayabrandi tips for how to do this?

File: tests/EPPs/files/test_buffer_exchange_twist_file.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                 Artifact(lims, id="2-2336706"),
+                 Artifact(lims, id="2-2336707"),
+                 Artifact(lims, id="2-2336708")]
+
+    # WHEN running get_file_data_and_write
+    get_file_data_and_write(artifacts, file)
+
+    # THEN the file has been created and missing udfs are replaced with the symbol "-".
+    assert file.read_text() == hamilton_buffer_exchange_no_udf
+    # remove file afterwards?
```
Comment (Karl-Svard) [inline tests/EPPs/files/test_buffer_exchange_twist_file.py:49]: @mayabrandi same here

**Review by mayabrandi — APPROVED** [review]
Looking good

**Review by mayabrandi — APPROVED** [review]
sweet

**Pull request merged**